tracer/src/Datadog.Trace.ClrProfiler.Native/rejit_handler.cpp

```diff
@@ -678,16 +682,23 @@ ULONG RejitHandler::ProcessModuleForRejit(const std::vector<ModuleID>& modules,
                               assemblyMetadata->version.str(), ").");
             }
 
-            // Check min version
-            if (integration.target_method.type.min_version > assemblyMetadata->version)
+            if (m_is_integration_version_checks_enabled)
             {
-                continue;
-            }
+                // Check min version
```

Comment:
I'm less convinced by this use case as - as someone pointed out - we usually check the min version compatibility when we code the integration. Though as it's behind a feature flag and as we check the parameters of the function anyway, it's clearly not a big deal.